hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/store/contracts/precompile/TransferPrecompilesTest.java

```diff
@@ -349,6 +353,22 @@ class TransferPrecompilesTest {
     private static final Bytes TRANSFER_NFTS_INPUT = Bytes.fromHexString(
             "0x2c4ba191000000000000000000000000000000000000000000000000000000000000047a000000000000000000000000000000000000000000000000000000000000008000000000000000000000000000000000000000000000000000000000000000e000000000000000000000000000000000000000000000000000000000000001400000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000047700000000000000000000000000000000000000000000000000000000000004770000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000047c000000000000000000000000000000000000000000000010000000000000047c0000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000007b00000000000000000000000000000000000000000000000000000000000000ea");
 
+    private enum WithHapiBlockLimit {
```

Comment:
Personally I think it should be left as is since it _really_ requires a different approach for coming up with these long hex strings.  They're basically the hex-encoding of the actual byte buffer passed to the contract, with selectors and parameters.  As a long hex string they're totally opaque - what method is it calling?  What are the arguments.   Did you stick the right number of `0`s of padding in there?  Dunno.  Can't tell you.  We need to have, as part of the BDD framework, _builders_ to build these things properly from their components.

Until then there's no point in merging these constant decls, and it might even impair actual readability.  Plus as is they serve to remind us to _fix_ this.

IMO.